### ISSUE TITLE
Expand "Anovos Workflow"

### ISSUE THREAD

**Issue by ionicsolutions** [anovos/anovos-docs#12]
This page would benefit from a concrete example from one of your teams. In the spirit of "sharing your knowledge and experience with the community", you could use this as an opportunity to share tips and best practices.

The workflow diagram is very complex and hard to follow. I suggest to break it down and/or focus on just the most relevant steps.

**Comment by kajanansangar** [comment]
Sure, Killian we are working on it.
